test/lib/usd/translators/UsdExportUVSetMappingsTest/UsdExportUVSetMappingsTest.ma

```diff
@@ -148,7 +173,7 @@ createNode transform -n "pPlane3";
 	rename -uid "A9516DAE-46FB-C555-7036-F5BFA863699B";
 	setAttr ".t" -type "double3" 2.4 0 0 ;
 	setAttr ".r" -type "double3" 89.999999999999986 0 0 ;
-createNode mesh -n "pPlaneShape3" -p "pPlane3";
+createNode mesh -n "pPlane3" -p "|pPlane3";
```

Comment:
@dgovil , @ysiewappl This makes the shape name identical to the transform name and makes sure the code in _UVMappingManager is still correctly handling that situation.